From 36c0d8791e3d550d16eb3599cc8e453b7e5dc4bc Mon Sep 17 00:00:00 2001
From: Guilherme Amadio <amadio@ncc.unesp.br>
Date: Mon, 30 Nov 2015 18:41:15 +0100
Subject: [PATCH] Fix search path for Intel compiler

---
 cmake/FindMIC.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/FindMIC.cmake b/cmake/FindMIC.cmake
index 875af21..d25c047 100644
--- a/cmake/FindMIC.cmake
+++ b/cmake/FindMIC.cmake
@@ -59,9 +59,9 @@ set(MIC_OFFLOAD_FOUND false)
 
 option(ENABLE_MIC "Enable native builds for the MIC architecture (Intel Knights Corner)" ON)
 if(ENABLE_MIC)
-   file(GLOB _intel_dirs "/opt/intel/compilers_and_libraries_*/linux")
+   file(GLOB _intel_dirs "@EPREFIX@/opt/intel/compilers_and_libraries_*/linux")
    if ("${_intel_dirs}" STREQUAL "")
-      file(GLOB _intel_dirs "/opt/intel/composer_xe_*")
+      file(GLOB _intel_dirs "@EPREFIX@/opt/intel/composer_xe_*")
    endif()
 
    list(SORT _intel_dirs)
-- 
2.6.4

