From 72d48a122ddc83852089219eefb8108d24a89277 Mon Sep 17 00:00:00 2001
From: Guilherme Amadio <amadio@ncc.unesp.br>
Date: Fri, 10 Apr 2015 17:16:12 -0500
Subject: [PATCH] Add option to turn on/off MIC build (default=on)

---
 CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 23b0472..8c5b9a5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,7 +17,12 @@ include (AddTargetProperty)
 include (OptimizeForArchitecture)
 
 vc_determine_compiler()
-find_package(MIC)
+
+option(BUILD_MIC "Build Vc library for MIC architecture" ON)
+
+if (BUILD_MIC)
+   find_package(MIC)
+endif()
 
 if(ROOT_RELEASE)
    if(EXISTS "${CMAKE_INSTALL_PREFIX}/Module.mk")
-- 
2.3.5

